### ISSUE TITLE
[BUG] Call to undefined method Illuminate\Support\Facades\DB::extend()

### ISSUE THREAD

**Issue by gabrielchavezme** [laraform/laraform#37]
### Prerequisites

* [x ] Able to reproduce the behaviour outside of your code, the problem is isolated to Laraform.
* [ x] Provided all the required dependencies to reproduce the bug (eg. form, component, model, migration).
* [ x] Your issue isn't already filed.
* [ x] You filled in the entire issue template.

### Versions

* PHP version: 7.3
* Laravel version: 6.9
* Laraform Vue package version: 1.1.8
* Laraform Laravel package version: 1.1

### Description

There is a problem related to multiple databases. If MongoDB is used with Laravel, it throws the following error: https://flareapp.io/share/47qGqBmW

If I remove the jenssegers / mongodb package, the problem is solved, but if I reinstall it, the problem returns.

### Steps to Reproduce
Install jenssegers/mongodb package composer

### Additional Information

The error appears to occur when the Mongodb provider is registered in the jenssegers / mongodb package

If in my class that extends Laraform I remove the variable or the schema method, the problem is deleted, but a Laraform error already throws me straight because I did not find that method

![Screenshot from 2019-12-28 01-01-25](https://user-images.githubusercontent.com/28960114/71539662-976c6b00-290d-11ea-9a68-6e8767d3a874.png)

**Comment by laraform** [comment]
Fixed in 1.1.9, please upgrade:
`composer require laraform/laraform-laravel`

**Comment by gabrielchavezme** [comment]
I have updated laraform to the latest version and the problem continues.

In my controller I have removed the variable:

`$form = resolve('App\Forms\ConfidenceContacts');`

And stop the error, apparently it is a dependency error when the global helper resolve () is called.


I attach errors uploaded to flare for a more detailed debug: https://flareapp.io/share/o7AnOJ5p#F85

I tried to solve the error but I could not, apparently butt wall.
